bci_tester/data.py

```diff
@@ -63,15 +70,28 @@
     OS_SP_VERSION = 0
     OS_CONTAINER_TAG = "latest"
     APP_CONTAINER_PREFIX = "opensuse"
+    BCI_CONTAINER_PREFIX = "bci"
 
     #: The Tumbleweed pretty name (from /etc/os-release)
     OS_PRETTY_NAME = os.getenv(
         "OS_PRETTY_NAME",
         "openSUSE Tumbleweed",
     )
+elif OS_VERSION == "basalt":
+    OS_MAJOR_VERSION = 16
```

Comment:
Has this been agreed to be the case now?

Reply:
this is what we use as %suse_version. I don#t know how official it is.